addons/website_event_stand/controllers/main.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http, _
+from odoo.http import request
+from odoo.addons.website_event.controllers.main import WebsiteEventController
+from werkzeug.exceptions import NotFound
+
+
+class WebsiteEventStandController(WebsiteEventController):
+
+    @http.route(['/event/<model("event.event"):event>/stands/register'],
+                type='http', auth='public', website=True, sitemap=False)
+    def event_stand_register(self, event, **kwargs):
+        values = {
+            'event': event,
+            'main_object': event,
+        }
+        return request.render('website_event_stand.event_stand_registration', values)
+
+    @http.route(['/event/<model("event.event"):event>/stands/confirm'],
```

Comment:
Careful, there was an issue if we buy multiple slot at once, at will not add the `extra_price`
(I know you know it, just to not forget :) )